net.tangysoft.usenext.yaml

```diff
@@ -0,0 +1,64 @@
+app-id: net.tangysoft.usenext
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: '46'
```

Comment:
gnome 47 is out.
```suggestion
runtime-version: '47'
```

Reply:
but do you really need the gnome runtime?

Reply:
maybe not. I can try the freedesktop SDK. The app does not have many requirements. It's a Web app, after all.

FWIW: It doesn't work with 47; gtk2 doesn't build IIRC.

Reply:
```
gtkcellrendererspin.c:210:26: error: assignment to ‘GtkAdjustment *’ {aka ‘struct _GtkAdjustment *’} from incompatible pointer type ‘GObject *’ {aka ‘struct _GObject *’} [-Wincompatible-pointer-types]
  210 |         priv->adjustment = g_object_ref_sink (obj);
      |                          ^
```

Reply:
if you use the lates shared modules all the patch are there.